superset/views/database/forms.py

```diff
@@ -239,54 +241,7 @@ def at_least_one_schema_is_allowed(database: Database) -> bool:
     )
 
 
-class ExcelToDatabaseForm(DynamicForm):
-    # pylint: disable=E0211
-    def excel_allowed_dbs() -> List[Database]:  # type: ignore
-        # TODO: change allow_csv_upload to allow_file_upload
```

Comment:
This is a longstanding TODO so I do think we should go ahead with the db migration.